minesweeper/use-cases/startGame/startGame.ts

```diff
@@ -0,0 +1,67 @@
+import { BoardModel } from '@minesweeper/domain/Board.model'
+import { BombModel } from '@minesweeper/domain/Bomb.model'
+import { CellModel } from '@minesweeper/domain/Cell.model'
+import { FlagModel } from '@minesweeper/domain/Flag.model'
+import { NeighborsBombsCounter } from '@minesweeper/domain/NeighborsBombsCounter.model'
+
+interface IStartGameProps {
+	rows: number
+	columns: number
+	bombs: number
+}
+
+type IBoardResponse = BoardModel
+
+type IStartGameUseCase = (props: IStartGameProps) => IBoardResponse
+
+/**
+ * start game base on initial props
+ */
+
+export const startGameUseCase: IStartGameUseCase = ({
+	bombs: bombsInput,
+	rows,
+	columns,
+}) => {
+	if (bombsInput < 1) {
+		throw new Error('bombs must be greater than 0')
```

Comment:
I would put that errors in an Enum

enum Error {
  BOMBS_GREATER_THAN_ZERO = ''bombs must be greater than 0'',
  // each case
}

Reply:
I agree